service/dap/daptest/client.go

```diff
@@ -173,9 +173,11 @@ func (c *Client) ExpectOutputEventTerminating(t *testing.T) *dap.OutputEvent {
 	return c.ExpectOutputEventRegex(t, `Terminating process [0-9]+\n`)
 }
 
-func (c *Client) ExpectOutputEventClosingClient(t *testing.T) *dap.OutputEvent {
+const ClosingClient = "Closing client session, but leaving multi-client DAP server at .+:[0-9]+ with debuggee %s\n"
```

Comment:
Do we need to export this?

Reply:
They are used directly in some server_test tests in the dap package. I can add helpers to hide them if you think it's worth it.

Reply:
I mean additional helpers.